src/library/Uno.Material/Styles/Application/v2/Typography.xaml

```diff
@@ -1,177 +1,421 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
 
-	<!-- COMMON TEXT VARIABLES FOR v2 -->
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Light">
 
-	<!-- IMPORTANT NOTE :
+			<!-- COMMON TEXT VARIABLES FOR v2 -->
+
+			<!-- IMPORTANT NOTE :
+	CharacterSpacing is measured in 1/1000 of an "em". One "em" is equivalent to the current font size of the control.
+	For example, here with LabelMedium an additional 12 pixels will be inserted between each character with FontSize="12" and CharacterSpacing="1000".
+	So in order to specify the letter spacing of 0.5px from the M3  guidelines, we will need to add CharacterSpacing="41.666".
+	To resume: CharacterSpacing = (letter spacing or tracking value / FontSize) * 1000 -->
+
+			<!-- DISPLAY -->
+
+			<!--  Display Large  -->
+			<x:String x:Key="DisplayLargeFontWeight">Normal</x:String>
+			<x:Double x:Key="DisplayLargeFontSize">57</x:Double>
+			<!--  Tracking: -0.25 px -->
+			<!--  Diverging from Material M3 guidelines value. Source of truth Uno Figma file  -->
+			<!--  Compromise between the official M3 and Uno's Figma M3 design to ensure compatibility with all design systems styles  -->
+			<x:Int32 x:Key="DisplayLargeCharacterSpacing">-17.857</x:Int32>
```

Comment:
while here, we might want to correct this
using double literal as value for int is not good
the decimal parts are discard anyways since the receiving dp is expecting `int`